common-design/css/styles.css

```diff
@@ -935,17 +936,18 @@ button.cd-user-menu__item {
   padding-bottom: 48px; }
 
 @media (min-width: 768px) {
-  .nocssgrid.no-cssgridlegacy .cd-footer {
-    padding-bottom: 48px; }
   .cd-footer__inner {
-    display: -ms-grid;
-    display: grid;
-    -ms-grid-columns: 1fr 12px 1fr;
-    -ms-grid-rows: auto 48px auto 48px auto;
-    grid-template-columns: repeat(2, 1fr);
-    grid-gap: 12px;
-    grid-row-gap: 48px; }
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
```

Comment:
@rupl I notice I am not getting any -ms- prefixes for flexbox. I believe -ms-grid etc was manually added in the previous version

Reply:
if autoprefixer decides it's not necessary, then we believe autoprefixer 😄 — the compatibility library is based on caniuse data which is potentially updated on anytime `npm i` is run. So the results of autoprefixer can change even when it seems like nothing changed.